index.html

```diff
@@ -26,44 +39,47 @@
     <!-- endbower -->
     <script id="view-welcome" type="x-tmpl-mustache">
       <div class='alert errors-schedule'>
-        <ul>
-        </ul>
       </div>
+
       <form class='create-email'>
 
       <div class = "email-create">
-        <div>
-          <h2>New Schedule</h2>
+        <div class = "sending">
           <label>
-            <input name="email[to]" type='email' placeholder="To" />
+            <input name="email[to]" type='email' class = "send-to" placeholder=" To:" style="font-family:Arial, FontAwesome" />
```

Comment:
`style` attributes are used directly on html elements if they don't need to be. It would be better to style this in the CSS.